_includes/report.html

```diff
@@ -0,0 +1,34 @@
+<section>
+  <header class="major">
+    <h2>{{ site.data['locales'][page['lang']]['sections']['translation_report'] }}</h2>
```

Comment:
page var doesn't contain this info somewhere?

Reply:
Not that I could find.  These pages aren't processed by the Elixir School generator so they're lacking all of the data it generates.

Reply:
Oh.. it makes sense then